### DISCUSSION TITLE
How to enable podman push logs on Mac?

### DISCUSSION THREAD

**Discussion by avsthiago** [containers/podman#13809] [Q&A] [answered]
Hi 👋 

I'm pushing some large images to ECR using podman from Mac, and I'm missing the progress bars during the upload. There's no output while running the command. 

I tried to set `--log-level=debug`, but the last message that I get is:
```
DEBU[0000] DoRequest Method: POST URI: http://d/v4.0.3/libpod/images/imagename:0.7.1/push
```

I'm running podman `v4.0.3`, but the same was happening in the `v3.4.4`.

Could you help me finding where to enable the logs?

**Comment by Luap99** [comment, marked as answer]
I don't think is implemented: https://github.com/containers/podman/issues/11554

**Reply by avsthiago** [reply]
Thank you @Luap99!
